### PR TITLE
app-text/xml2rfc: new package, add 3.15.0 

### PR DESCRIPTION
**Pull request by Flowdalic**

### CODE REVIEW

**Comment by Flowdalic** [comment]
I've submitted a [PR that drops xml2rfc's dependency on kitchen](https://github.com/ietf-tools/xml2rfc/pull/913). Let's see if this gets through, so we don't have to re-introduce kitchen to the tree for xml2rfc.

File: dev-python/dict2xml/dict2xml-1.7.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+DESCRIPTION="Small utility to convert a python dictionary into an XML string"
+HOMEPAGE="https://pypi.org/project/dict2xml/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# The dict2xml archive from PyPi doesn't have the tests included :(
```
Comment (thesamesam) [inline dev-python/dict2xml/dict2xml-1.7.1.ebuild:18]: Use gh instead?

File: dev-python/dict2xml/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<name>Florian Schmaus</name>
+		<email>flow@gentoo.org</email>
+	</maintainer>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+	</maintainer>
+	<upstream>
+		<remote-id type="pypi">kitchen</remote-id>
```
Comment (thesamesam) [inline dev-python/dict2xml/metadata.xml:12]: Fix + add gh if it exists? And ALLARCHES?

File: dev-python/google-i18n-address/google-i18n-address-2.5.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+DESCRIPTION="Google's i18n address metadata repository"
+HOMEPAGE="https://pypi.org/project/google-i18n-address/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (thesamesam) [inline dev-python/google-i18n-address/google-i18n-address-2.5.2.ebuild:17]: Drop?

File: dev-python/google-i18n-address/google-i18n-address-2.5.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
```
Comment (thesamesam) [inline dev-python/google-i18n-address/google-i18n-address-2.5.2.ebuild:6]: PEP517?

File: dev-python/google-i18n-address/metadata.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<name>Florian Schmaus</name>
+		<email>flow@gentoo.org</email>
+	</maintainer>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+	</maintainer>
+	<upstream>
+		<remote-id type="pypi">google-i18n-address</remote-id>
```
Comment (thesamesam) [inline dev-python/google-i18n-address/metadata.xml:12]: gh remote-Id + ALLARCHES?

File: dev-python/google-i18n-address/google-i18n-address-2.5.2.ebuild (hunk 3 above)
Comment (thesamesam) [inline dev-python/google-i18n-address/google-i18n-address-2.5.2.ebuild:17]: Also, tests? Or do none exist?
